packages/lit-analyzer/src/analyze/default-lit-analyzer-context.ts

```diff
@@ -4,6 +4,15 @@ import * as tsServer from "typescript/lib/tsserverlibrary";
 import { analyzeComponents, analyzeLibDomHtmlElement } from "web-component-analyzer";
 import { getBuiltInHtmlCollection } from "./data/get-built-in-html-collection";
 import { getUserConfigHtmlCollection } from "./data/get-user-config-html-collection";
+import noBooleanInAttributeBindingRule from "../rules/no-boolean-in-attribute-binding";
```

Comment:
we could have a `rules/all.ts` or `index.ts` which exports each one, then `import * as rules from "../rules/index"` instead to already have an object of `{[key: string]: RuleModule}`